docs/components/vxp-pagination.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebarDepth: 0
+---
+
+# VxpPagination
+
+This is an overview of the most common usage of `<VxpPagination>`.
+`VxpPagination` is a UI component that displays page number for the given properties.
+
+## Live Demo
+<DocExampleBox :liveDemoMode="true">
+```html
+
+```
+<VxpPaginationLiveDemo />
+</DocExampleBox>
+
+## Props
+
+| Name             | Type    | Default | Web | Mobile | Description |
+| ---------------- | ------- | ------- | --- | ------ | ----------- |
+| totalItems       | Number  | `-`     | ✔   | ✔      | Total number of items in the list that is paginated. |
+| itemsPerPage     | Number  |  10     | ✔   | ✔      | Number of items that are displayed in a page |
+| page             | Number  |  1      | ✔   | ✔      | Selected page in pagination component. |
```

Comment:
it may be good to tell that possible values of page attribute starts from *1*

Reply:
updated the doc